recipes/astrometry/build.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+# Remove ncurses tic.h that conficts with astrometry tic.h
+rm -f $CONDA_PREFIX/include/tic.h
```

Comment:
You shouldn't remove files from the existing environment and the conflict could make a horrible mess to debug (i.e. "What's wrong with this header, it's right there!"). Why do you need to?

Reply:
Conda `ncurses` package put a `tic.h` file in `$PREFIX/include`. `astrometry` have a [file](https://github.com/dstndstn/astrometry.net/blob/588ccb289983b615f0b784354114e7d7550c8e45/include/astrometry/tic.h) named `tic.h` in a folder. When the program tries to load this file, it fails to compile in conda environment if this file isn't removed, like in [1](https://github.com/dstndstn/astrometry.net/blob/555b6ba637d19a8c35b7402fbc85499fb5db1945/blind/index-info.c#L16), [2](https://github.com/dstndstn/astrometry.net/blob/555b6ba637d19a8c35b7402fbc85499fb5db1945/util/tic.c#L14), [3](https://github.com/dstndstn/astrometry.net/blob/555b6ba637d19a8c35b7402fbc85499fb5db1945/blind/blind-main.c#L13) and [4](https://github.com/dstndstn/astrometry.net/blob/555b6ba637d19a8c35b7402fbc85499fb5db1945/blind/hpquads-main.c#L24).

This may be a wrong priority order in make file, but I had to add this line to make the program compile in any conda environment (local, local ubuntu docker, conda-forge docker local, conda-forge ci).